resources/manifest_schema.json

```diff
@@ -0,0 +1,156 @@
+{
+  "anyOf": [
+    {
+      "type": "array",
+      "minItems": 1,
+      "items": {
+        "$ref": "#/project"
+      }
+    },
+    {
+      "$ref": "#/project"
+    }
+  ],
```

Comment:
took me some time to find but i think it's cool to 
there is a little issue when tracking errors with this one:
when ever u find an error while parsing it throws two errors, one is that is doesn't match the first schema provided and the other error is the actual error
kind of confusing but im not sure if we should keep it that way or try to hide the error for "not an array" if its not an array anyways